Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -6330,6 +6331,14 @@ float LocalFrameView::deviceScaleFactor() const
     return 1;
 }
 
+void LocalFrameView::writeRenderTreeAsText(TextStream& ts, OptionSet<RenderAsTextFlag> behavior)
+{
+    auto* localFrame = dynamicDowncast<LocalFrame>(frame());
+    if (!localFrame)
+        return;
+    ts << externalRepresentationForLocalFrame(localFrame, behavior, ts.indent());
```

Comment:
This adds an unnecessary String allocation that wasn't there before.  Can we pass ts as a parameter to the common function, then create a TextStream in WebProcess::renderTreeAsText?